examples/nlp/language_modeling/megatron_gpt_eval.py

```diff
@@ -199,7 +199,7 @@ def main(cfg) -> None:
 
     assert (
         cfg.trainer.devices * cfg.trainer.num_nodes
-        == cfg.tensor_model_parallel_size * cfg.pipeline_model_parallel_size
+        == cfg.tensor_model_parallel_size * cfg.pipeline_model_parallel_size * max(1, cfg.expert_model_parallel_size)
```

Comment:
do you want to use cfg.get instead, otherwise might break on checkpoints without EP?